docs/getting_started.md

```diff
@@ -0,0 +1,170 @@
+# Getting started
+
+These quickstarts run on WildFly 10.
+
+Prior to running the quickstarts you should read this entire document and have completed the following steps:
+
+* [Start and configure the Keycloak Server](#keycloak)
+* [Start and configure the WildFly Server](#wildfly)
+
+Afterwards you should read the README file for the quickstart you would like to deploy. See [examples](#examples) for
+a list of the available quickstarts.
+
+If you run into any problems please refer to the [troubleshooting](#troubleshooting) section.
+
+
+## Use of KEYCLOAK\_HOME and WILDFLY\_HOME Variables
+
+The quickstart README files use the replaceable value KEYCLOAK\_HOME to denote the path to the Keycloak installation and the
+value WILDFLY\_HOME to denote the path to the WildFly installation. When you encounter this value in a README file, be sure
+to replace it with the actual path to your installations.
+
+
+## System Requirements
+
+The applications these projects produce are designed to be run on WildFly Application Server 10.
+
+All you need to build these projects is Java 8.0 (Java SDK 1.8) or later and Maven 3.1.1 or later.
+
+
+## <a id="keycloak"></a>Start the <span>Keycloak</span> Server
+
+By default the Keycloak Server uses the same ports as the WildFly Server. To run the quickstarts you can either run the
+ Keycloak Server on a separate host (machine, VM, Docker, etc..) or on different ports.
+
+To start the Keycloak server on a separate host:
+
+1. Open a terminal on the separate machine and navigate to the root of the Keycloak server directory.
+
+2. The following shows the command to start the Keycloak server:
+
+   ````
+   For Linux:   KEYCLOAK_HOME/bin/standalone.sh -b 0.0.0.0
+   For Windows: KEYCLOAK_HOME\bin\standalone.bat -b 0.0.0.0
+   ````
+
+3. The URL of the Keycloak server will be http://&lt;HOSTNAME&gt;:8080 (replace &lt;HOSTNAME&gt; with the hostname of the separate host).
+
+To start the Keycloak server on different ports:
+
+1. Open a terminal and navigate to the root of the Keycloak server directory.
+
+2. The following shows the command to start the Keycloak server:
+
+   ````
+   For Linux:   KEYCLOAK_HOME/bin/standalone.sh -Djboss.socket.binding.port-offset=100
+   For Windows: KEYCLOAK_HOME\bin\standalone.bat -Djboss.socket.binding.port-offset=100
+   ````
+
+3. The URL of the Keycloak server will be *http://localhost:8180*
```

Comment:
Is there any particular reason to have the address in italics?

Reply:
Not really, updated.